providers/base/bin/prime_offload_tester.py

```diff
@@ -0,0 +1,311 @@
+#!/usr/bin/env python3
+"""prime offload test module."""
```

Comment:
Please add author info like this:
'''
Copyright (C) 2023 Canonical Ltd.

Authors
  Hanhsuan Lee <hanhsuan.lee@canonical.com>
'''